### PR TITLE
1780-add-img-translation-tutorial

### LINKED ISSUE #1780: image_to_image_translation

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Fixes #1780 #1784 

### Description
A few sentences describing the changes proposed in this pull request.

### Checks

- [ ] Avoid including large-size files in the PR.
- [ ] Clean up long text outputs from code cells in the notebook.
- [ ] For security purposes, please check the contents and remove any sensitive info such as user names and private key.
- [ ] Ensure (1) hyperlinks and markdown anchors are working (2) use relative paths for tutorial repo files (3) put figure and graphs in the `./figure` folder
- [ ] Notebook runs automatically `./runner.sh -t <path to .ipynb file>`

### CODE REVIEW

File: generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb [hunk 1]
```diff
@@ -0,0 +1,1317 @@
+{
```
Comment (KumoLiu) [inline generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb:1]: <p>“## Preprocessing of the BRATS Dataset in 2D slices for training”.</p><p> </p><p>The first line has one more&nbsp;"\"</p><p> </p><p>Looks the description here is inconsistent with transforms below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (KumoLiu) [inline generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb:1]: <p>Please update the output for this cell, should be outdated here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (KumoLiu) [inline generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb:1]: <p>I would suggest move this cell to be in the next cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (KumoLiu) [inline generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb:1]: <p>please remove the output for this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (KumoLiu) [inline generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb:1]: <p>The level for the title seems not correct.</p><p>"## Model training of the Diffusion Model"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (KumoLiu) [inline generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb:1]: <p>We can include this cell with the cell below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (KumoLiu) [inline generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb:1]: <p>“## </p><p>Sampling of a new segmentation mask for an input image of the validation set”</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (KumoLiu) [inline generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb:1]: <p>please help remove the output for the network def here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (KumoLiu) [inline generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb:1]: <p>please help remove the "\" here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (yiheng-wang-nv) [reply generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb:1]: <p>It seems "\" is a special symbol in markdown, it's only used to change line, and will not show when open the notebook (not in this reivew NB)</p>

Comment (KumoLiu) [reply generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb:1]: Got it.

**Review by KumoLiu — APPROVED** [review]

**Pull request merged**